assemblies/dynamic-plugins/assembly-rhdh-installing-dynamic-plugins.adoc

```diff
@@ -33,12 +33,24 @@ include::../modules/dynamic-plugins/ref-rh-supported-plugins.adoc[leveloffset=+3
 // Technology preview support statement
 include::../../artifacts/snip-dynamic-plugins-support.adoc[leveloffset=+3]
 
-// Red Hat technology preview plugins
+// Red Hat Technology Preview plugins
+[id="rhdh-tech-preview-plugins"]
 include::../modules/dynamic-plugins/ref-rh-tech-preview-plugins.adoc[leveloffset=+4]
 
+[NOTE]
+====
+* A new Technology Preview plugin for Red Hat Ansible Automation Platform (RHAAP) is available, which replaces this older one. See link:{LinkPluginsGuide}#rhdh-compatible-plugins[Other compatible plugins] in the _{NameOfPluginsGuide} guide_ for further details.
+====
```

Comment:
We might want to keep this kind of information (previously ... currrently) only in release notes (for maintenance purpose).

Reply:
well, Ben's idea was to call out that there's a new plugin that will replace the old one, in situ with the list of plugins.